src/Symfony/Component/Yaml/Inline.php

```diff
@@ -146,6 +146,10 @@ public static function dump(mixed $value, int $flags = 0): string
             case false === $value:
                 return 'false';
             case \is_int($value):
+                if (Yaml::DUMP_NUMERIC_KEY_AS_STRING & $flags) {
```

Comment:
This will not only convert keys to String, also values will be converted: `[200 => 200]` will be `{ '200': '200' }`. I would not expect that by this flag name.